content/operate/rc/databases/connect.md

```diff
@@ -55,10 +55,10 @@ A Redis client is a software library or tool that enables applications to intera
 
 The connection wizard provides code snippets to connect to your database with the following programming languages:
 
-- node.js using [node-redis](https://github.com/redis/node-redis/blob/master/README.md)
+- node.js using [node-redis]({{< relref "/develop/connect/clients/nodejs" >}})
 - .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
-- Python using [redis-py](https://github.com/redis/redis-py#redis-py)
-- Java using [Jedis](https://github.com/redis/jedis#jedis)
+- Python using [redis-py]({{< relref "/develop/connect/clients/python/redis-py" >}})
+- Java using [Jedis]({{< relref "/develop/connect/clients/java/jedis" >}})
```

Comment:
Why not this link for C#/.Net ?

https://redis.io/docs/staging/DOC-4392/develop/connect/clients/dotnet/

Reply:
This is just listing the examples in the connection wizard, not the clients that Redis owns. @andy-stark-redis is working with the FE infrastructure team to change the examples in the connection wizard to those that are owned by Redis, but until that is live on production, the C#/.NET example is in StackExchange.Redis and not NRedisStack.

Reply:
Got it. Apologies for intruding.

Reply:
All good, it makes sense.